yb-voyager/cmd/importData.go

```diff
@@ -239,6 +239,9 @@ func fetchDefaultParllelJobs(targets []*tgtdb.Target) int {
 		}
 		totalCores += targetCores
 	}
+	if totalCores == 0 { //if target is running on MacOS, we are unable to determine totalCores
+		return 3
```

Comment:
Instead of a fixed constant `3`, can we have a better default like `2 * number of nodes` ?

Reply:
I believe we had decided during the standup meeting to keep it as a constant, and to keep it as 3- this case only arises when the target is running on MacOS (which is only possible when the user is running a local installation of YugabyteDB)